metaflow/metaflow_config.py

```diff
@@ -149,6 +136,27 @@ def from_conf(name, default=None):
 # Default container registry
 DEFAULT_CONTAINER_REGISTRY = from_conf("METAFLOW_DEFAULT_CONTAINER_REGISTRY")
 
+##
+# KFP configuration
+###
+# Link to image package:
+# https://github.com/zillow/metaflow/pkgs/container/metaflow%2Fmetaflow-zillow
+if DEFAULT_CONTAINER_REGISTRY and DEFAULT_CONTAINER_IMAGE:
+    KFP_CONTAINER_IMAGE = (
+        f"{DEFAULT_CONTAINER_REGISTRY.rstrip('/')}/{DEFAULT_CONTAINER_IMAGE}"
+    )
+else:
+    KFP_CONTAINER_IMAGE = "ghcr.io/zillow/metaflow/metaflow-zillow:2.2"
```

Comment:
should this be named: `KFP_DEFAULT_CONTAINER_IMAGE` ?

Reply:
renamed

Reply:
it no longer has `DEFAULT` in the name?

Reply:
Sorry! Forget to push before merging 😓 . Adding changes now.

Reply:
https://github.com/zillow/metaflow/pull/174